database/migrations/2025_03_25_085142_wiki_profiles_audience_nullable.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+            DB::statement('
+                ALTER TABLE `wiki_profiles` CHANGE COLUMN `audience` `audience` 
+                ENUM("narrow", "wide", "other")
```

Comment:
```suggestion
                ENUM("narrow", "wide", "other") NULL
```

this seems to fix it